.editorconfig

```diff
@@ -0,0 +1,38 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+# Set default charset
+charset = utf-8
+indent_style = space
+indent_size = 4
+
+# Follow https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md
```

Comment:
We already use CI (rust-fmt) to ensure this.

Reply:
yes.

It's better to have this or you need to run the rustFmt